src/Driver/driver_configs.jl

```diff
@@ -196,9 +214,12 @@ function AtmosLESConfiguration(
     numerical_flux_gradient = CentralNumericalFluxGradient(),
     fv_reconstruction = nothing,
     grid_stretching = (nothing, nothing, nothing),
+    Ncutoff = N,
```

Comment:
I'm testing this by running a different experiment than the `experiments/TestCase/isothermal_zonal_flow.jl` and this would break, if the default test arguments were used. I think we should give `Ncutoff` another default value, not the same as `N`.

Reply:
we could make it `get_polyorder(N)`?